ArduPlane/GCS_Mavlink.cpp

```diff
@@ -2067,6 +2067,51 @@ void GCS_MAVLINK::handleMessage(mavlink_message_t* msg)
         break;
     }
 
+    case MAVLINK_MSG_ID_SET_POSITION_TARGET_GLOBAL_INT:
+    {
+        mavlink_set_position_target_global_int_t pos_target;
+        mavlink_msg_set_position_target_global_int_decode(msg, &pos_target);
+        const uint16_t alt_mask = 0b0000000000000100; // (alt mask at bit 3)
+            
+        bool msg_valid = true;
+        AP_Mission::Mission_Command cmd = {0};
+
+        if(pos_target.type_mask ^ alt_mask)
```

Comment:
nit picky stuff here, but our coding standard has a space behind each keyword of if and switch. Also, please don't put brackets in case statements that don't need them, just makes the code longer to read with zero benefit. Also, is that type_mask/alt_mask thing supposed to be an & (and) like a usual bit mask?